pkg/security/secl/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/DataDog/datadog-agent/pkg/security/secl
 
-go 1.22.0
```

Comment:
Is this expected ?

Reply:
I don't think so, this was done automatically by `inv tidy`. This might be related to my env, I'll look into it.